### DISCUSSION TITLE
small explosion rework

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6295] [Ideas and suggestions]
i think it's kinda ridiculous and inconsistent how explosions multiply their damage on each limb they hit, what if it just divided instead? maybe buff most explosives to compensate, or reduce/remove damage falloff over radius (which is also inconsistent).

**Comment by Tsunder** [comment]
![image](https://user-images.githubusercontent.com/5103517/125212972-a7913480-e27e-11eb-9532-345d191a9c75.png)

**Reply by LePiromano** [reply]
the "after" is how it works right now, the 180 damage from the frag grenade multiplies on every limb dealing 1080 damage (if we're talking about a human)
